.travis.yml

```diff
@@ -14,4 +14,5 @@ cache:
     - $HOME/.gradle/wrapper/
 
 after_success:
-  - ./script/transifex.sh
\ No newline at end of file
+  - ./script/transifex.sh
+  - if [ "$TRAVIS_BRANCH" == "1.7.10-MNA" ]; then ./gradlew bintrayUpload; fi
```

Comment:
Deploying snapshots is a good idea.
If we upload snapshots  from the `1.7.10-MNA` branch, we should work on features branches only, on not any more from the main branch.